src/Sarif.UnitTests/TestData/InsertOptionalDataVisitor/CoreTests.sarif

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-2.0.0",
+  "version": "2.0.0",
+  "runs": [
+    {
+      "tool": { "name": "Sarif.UnitTests",
+        "fullName": "Sarif.UnitTests.1.3.3.0-beta", "version": "1.3.3.0", "semanticVersion": "1.3.3", "sarifLoggerVersion": "1.5.22.0", "language": "en-US"},
+      "originalUriBaseIds": { "SRCROOT": "file:///e:/src/sarif-sdk/" },
+      "results": [        
+          {"ruleId": "TEST2001", "message": { "text": "This result is at the beginning of the file and selects 'Region_At_Start_Of_File'." },
+          "locations": [{"physicalLocation": { "fileLocation": { "uri": "src/Sarif.UnitTests/TestData/InsertOptionalDataVisitor/InsertOptionalDataVisitor.txt", "uriBaseId": "SRCROOT" },
```

Comment:
>src/Sarif.UnitTests/TestData [](start = 73, length = 28)

You could make these files shorter by including this in the URI base id, and renaming it from `SRCROOT` to `TEST_DATA_ROOT`.